tutorial-week-5.md

```diff
@@ -0,0 +1,20 @@
+How to use SCSS and some of its tools
```

Comment:
This is a pretty broad topic! And you only discussed Variables. Try honing your topic more and focusing on one specific skill.